contracts/libraries/DataTypes.sol

```diff
@@ -39,23 +39,23 @@ library CoreTypes {
          *
          */
         uint8 channelState;
-        // @notice denotes the address of the verifier of the Channel
+        /// @notice denotes the address of the verifier of the Channel
         address verifiedBy;
-        // @notice Total Amount of Dai deposited during Channel Creation
+        /// @notice Total Amount of Dai deposited during Channel Creation
```

Comment:
Total Amount of PUSH (instead of DAI)

Reply:
This has now been fixed along with a few other minor fixes, merging changes now. cc: @Zartaj0